optimum/habana/transformers/models/starcoder2/modeling_starcoder2.py

```diff
@@ -151,6 +185,8 @@ def forward(
         output_attentions: Optional[bool] = False,
         use_cache: Optional[bool] = False,
         token_idx: Optional[torch.Tensor] = None,
+        use_flash_attention: Optional[bool] = False,
+        flash_attention_recompute: Optional[bool] = False,
```

Comment:
minor/consistency: Please update the only differences are section as you did on lines 58, 59 for the functions you added the arguments to.

Reply:
Added